src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx

```diff
@@ -153,4 +153,7 @@
   <data name="Regex_OtherEscape" xml:space="preserve">
     <value>Regex - Other Escape</value>
   </data>
+  <data name="Configure_or_Suppress_issues" xml:space="preserve">
+    <value>Configure or Suppress issues</value>
```

Comment:
casing is wonky.  Should be `Configure or suppress issues`.

Reply:
I have explicitly kept "Suppress" with capital S, given that we used to almost always have a top level item for Suppress that is now nested. I can change if you/others strongly feel otherwise.